bokeh/util/notebook.py

```diff
@@ -32,10 +32,6 @@ def load_notebook(resources=None, verbose=False, hide_banner=False):
 
 FINALIZE_JS = """
 Bokeh.$("#%s").text("BokehJS successfully loaded");
```

Comment:
This should probably change to somethign like `BokehJS loading...` and then if there's a way to add a `done.` at the end after some timeout / check, great.